pkg/runtime/wfengine/backends/actors/actors.go

```diff
@@ -72,6 +72,7 @@ type Options struct {
 
 type Actors struct {
 	appID             string
+	namespace         string
```

Comment:
Remove

Reply:
I cannot. This is 100% needed so that I know of the namespace when building out the `wfActorType = common.GetWorkflowActorType(a.namespace, router.GetSource())` inside `pkg/actors/targets/workflow/activity/execute.go` in the `executeActivity` func under the commented line for `// send completed event to orchestrator wf actor` bc namespace is needed when constructing the `wfActorType`